jc/parsers/ascii.py

```diff
@@ -0,0 +1,178 @@
+"""jc - JSON CLI output utility ASCII table parser
+
+Supports common ASCII table formats
+
+Usage (cli):
+
+    $ virsh list --all | jc --ascii
+
+    or
+
+    $ jc ascii
+
+Usage (module):
+
+    import jc.parsers.ascii
+    result = jc.parsers.ascii.parse(ascii_table_output)
+
+Schema:
+
+    [
+      {
+        "column1":     string,
+        "column2":     boolean,
+        "column3":     integer
+      }
+    ]
+    
+Examples:
+
+    $ virsh domifaddr foo | jc --ascii
+    [{"name":"vnet0","mac_address":"01:23:45:67:89:ab","protocol":"ipv4","address":"192.168.122.100/24"}]
+
+"""
+import jc.utils
+import jc.parsers.universal
+import re
+
+
+class info():
+    """Provides parser metadata (version, author, etc.)"""
+    version = '1.0'
+    description = 'ASCII table parser'
+    author = 'Alexandru Barbur'
+    author_email = 'alex@ctrlc.name'
+    # details = 'enter any other details here'
+
+    # compatible options: linux, darwin, cygwin, win32, aix, freebsd
+    compatible = ['linux', 'darwin', 'cygwin', 'win32', 'aix', 'freebsd']
+    magic_commands = ['ascii']
```

Comment:
You can go ahead and remove/comment out the `magic_commands` attribute since it is only used for magic syntax and there is no command for `jc` to run and use this parser.